src/audio/detect_test.c

```diff
@@ -659,8 +697,21 @@ static int test_keyword_reset(struct comp_dev *dev)
 
 static int test_keyword_prepare(struct comp_dev *dev)
 {
+	struct comp_data *cd = comp_get_drvdata(dev);
+	uint16_t valid_bits = dev->params.sample_valid_bytes * 8;
+	uint16_t sample_width = cd->config.sample_width;
+
 	trace_keyword("test_keyword_prepare()");
 
+	if (valid_bits != sample_width) {
+		/* Default threshold value has to be changed
+		 * according to host new format.
+		 */
+		cd->config.activation_threshold = (valid_bits > 16U) ?
```

Comment:
Here you are using 16U and earlier just 16. Lack of consistency.

Reply:
Here it is correct, I will force the constant to be unsigned in that other place too. Thanks for spotting.